src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-details/pool-details.component.html

```diff
@@ -9,7 +9,7 @@
          i18n>Details</a>
       <ng-template ngbNavContent>
         <cd-table-key-value [renderObjects]="true"
-                            [data]="filterNonPoolData(selection)"
+                            [data]="poolDetails"
```

Comment:
Nice to know and in many cases a good first approach, but the selection is updated through a new assigned every 5 seconds (which triggers `ngOnChanges` every time), if specific details were not filtered out, like they are now, the details would be unequal every time, because of the time series data.

Reply:
If we don't want to trigger change detection when nothing really changes, we could filter streams through [`distinct*` operators](https://www.learnrxjs.io/learn-rxjs/operators/filtering/distinctuntilchanged)... or use OnPush strategy. On the time series, I find those not very friendly in tabular mode... wouldn't it make sense to remove it/hide it from the details table?

Reply:
They are removed from the seen details table in this PR but the selection that is passed from the pool table still has the properties in the selection object, therefore every selection object differs.

Reply:
With the `comparator` function you can instruct the `distinctUntilChanges` operator to distinguish when an object really changes.

Reply:
I tried that but it looked kind of strange creating a observable from something I don't get as observable in the details component than pipeing it through `distinctUntilChanges` with the omitted the keys from the keys the object has to subscribe to the change to assign the changed, rather than just do what I did use omitting the keys from the object (which reference always changes) (with `_.omit` and checking if its values differ (with `_.isEqual`), if yes assign the changed.

I also tried the push strategy and that worked great :+1:  thanks for that tip :)